beetsplug/spotify.py

```diff
@@ -674,13 +677,22 @@ def _fetch_info(self, items, write, force):
             if write:
                 item.try_write()
 
-    def track_popularity(self, track_id=None):
-        """Fetch a track popularity by its Spotify ID."""
+    def track_info(self, track_id=None):
+        """Fetch track popularity and other external_ids using its Spotify ID."""
```

Comment:
The reason I corrected this sentence is that it was, and now still is, wrong in terms of content. It means this:

We fetch an ID named track popularity and we also fetch *other* external ID's. -> That's not true.

To make this sound correct you at least have to remove the word "other" in this sentence (if you don't like my suggested longer version)

Sorry for being picky here. Even it is just a developer's comment, I'd still consider this "documentation" and thus I'm worried about correctness a lot :-)